### PR TITLE
New in php7.*. Minor text adjustments

### PR DESCRIPTION
**Pull request by a-kharunzha**

### CODE REVIEW

File: PHP/Language.md [hunk 1]
```diff
@@ -84,6 +94,10 @@ Validate (in_array(), array_key_exists() etc)
 
 Sort and shuffle arrays
 
+Spread Operator in Array Expression
+
+Unpacking arrays to variables using list(). By value and by reference difference.
```
Comment (AndriyHudyma) [inline PHP/Language.md]: Please check and correct this sentence: "By value and by reference difference"

File: PHP/Language.md [hunk 2]
```diff
@@ -154,29 +174,38 @@ $this vs self
 
 static properties and methods
 
-Magic methods (constructors, object as callable, overloading etc)
+Magic methods 
+* constructors, 
+* overloading methods and properties
+* object as callable
+* serialization and json encoding control
+     __serialize/__unserialize. vs __sleep/__wakeup
```
Comment (AndriyHudyma) [inline PHP/Language.md:182]: __serialize/__unserialize are not magic methods, right?

File: PHP/Language.md [hunk 3]
```diff
@@ -263,7 +292,7 @@ Specifying a class with bindTo()
 
 Predefined Interfaces:  ArrayAccess, Serializable, Traversable, Iterator and IteratorAggregate
 
-Generators
+Generators. yield, yield from, gerReturn()
```
Comment (AndriyHudyma) [inline PHP/Language.md]: ::getReturn

File: PHP/Language.md (hunk 2 above)
Comment (a-kharunzha) [reply PHP/Language.md:182]: They are in php7.4. https://wiki.php.net/rfc/custom_object_serialization

File: PHP/Language.md (hunk 3 above)
Comment (a-kharunzha) [reply PHP/Language.md]: I did not add `::` at first to not mix up with static methods. But for showing that it **is** a method, I'll add it.

**Pull request merged**